build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts

```diff
@@ -16,7 +16,20 @@
 
 repositories {
     maven {
-        name = "Gradle libs"
-        url = uri("https://repo.gradle.org/gradle/libs")
+        name = "Gradle public repository"
+        url = uri("https://repo.gradle.org/gradle/public")
+        content {
+            includeGroup("net.rubygrapefruit")
+            includeModule("classycle", "classycle")
+            includeModule("flot", "flot")
+            includeModuleByRegex("org.gradle", "sample-(check|discovery)")
+        }
     }
+    google {
```

Comment:
❓ why do we control which artifacts come from which repositories?

Reply:
We know that certain artifacts are only availble in certain repositories. So it makes no sense to query other repositories for them. Our strategy is to have few repositories that provide "special" artifacts like repo.gradle.org or the goolge repository. For those repositories we filter what we pull from there. Everything else will be resolved via maven central.